kong/kong.lua

```diff
@@ -1,4 +1,4 @@
--- Kong, the biggest ape in town
+-- Kong, the biggest ape in town 1
```

Comment:
`git commit --allow-empty` can be neat for that, avoiding to do painful little changes to force one.